lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -808,15 +808,23 @@ struct StoreOpConversion
     auto valueElems = getLLVMElems(value, llValue, layout, rewriter, loc);
     assert(ptrElems.size() == valueElems.size());
 
+    // Determine the vectorization size
+    size_t vec = getVectorizeSize(ptr, layout);
     SmallVector<Value> maskElems;
     if (llMask) {
       maskElems = getLLVMElems(mask, llMask, layout, rewriter, loc);
       assert(valueElems.size() == maskElems.size());
+      auto maskOrder = mask.getType()
+                           .cast<RankedTensorType>()
+                           .getEncoding()
+                           .cast<BlockedEncodingAttr>()
```

Comment:
For `StoreOp`, it is for sure that only BlockedEncodingAttr will be used here?

Reply:
Currently, the StoreOp only works in blocked layout. If we need to extend with other layouts, it will yield a fatal here and remind to update the code.